Makefile.js

```diff
@@ -73,14 +82,26 @@ target.lint = function() {
         errors++;
     }
 
+    echo("Validating CJS JavaScript test files");
+    lastReturn = exec(ESLINT + CJS_TEST_FILES);
+    if (lastReturn.code !== 0) {
+        errors++;
+    }
+
     if (errors) {
         exit(1);
     }
 };
 
 target.test = function() {
     let errors = 0;
-    const lastReturn = exec(`${ISTANBUL} cover ${MOCHA} -- -R progress -c ${TEST_FILES}`);
+    let lastReturn = exec(`${MOCHA} -- -R progress -c ${CJS_TEST_FILES}`);
```

Comment:
```suggestion
    let lastReturn = exec(`${NODE} ${MOCHA} -- -R progress -c ${CJS_TEST_FILES}`);
```

To make it work on Windows.